todo_app/app.py

```diff
@@ -5,35 +5,47 @@
 from todo_app.data.session_items import add_item
 from todo_app.data.session_items import get_item
 from todo_app.data.session_items import save_item
+from todo_app.data.trello_items import trello_get_items
+from todo_app.data.trello_items import trello_add_item
+from todo_app.data.trello_items import trello_complete_item
+from todo_app.data.trello_items import trello_todo_item
+from datetime import datetime
+import requests
+import os
+
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return render_template('index.html', result = get_items())
+
+    return render_template('index.html', result = trello_get_items())
 
 @app.route('/add', methods=['POST'])
 def add():
-    error = None
-    if request.method == 'POST':
-        if not request.form['title'].isspace():
-            add_item(request.form['title'])
-            return render_template('index.html', success="successfully written", result = get_items())
-        else:
-            error = 'no title'
-    # the code below is executed if the request method
-    # was GET or the credentials were invalid
-    return render_template('index.html', success="error ", result = get_items())
+    if not request.form['title'].isspace():
+        add_result = trello_add_item(request.form['title'], request.form['desc'], request.form['datepicker'])
+        return render_template('index.html', success=add_result, result = trello_get_items())
+    else:
+        return render_template('index.html', success="No title given", result = trello_get_items())
```

Comment:
`success` is a surprising variable name since it can be for error messages too. We don't want surprises.

How about `result_message`?

Also `result` is a slightly confusing name because it's not exactly the result of any user action. You could just call it e.g. `items`.